.github/workflows/changelog-assembly.yml

```diff
@@ -29,7 +29,8 @@ jobs:
         with:
           assignees: |
             AmmarAbouZor
-          branch: documentation/scriv-assemble-changelog
+          branch: documentation/aeruginous-ronlog
```

Comment:
This workflow does not use Scriv, at the moment, so I renamed the branch name as well as the job name to avoid confusion.